README.md

```diff
@@ -1,78 +1,74 @@
-  * swap the value of `application.gecko.id` (e.g. `ion-basic-study@mozilla.org`) with the one provided you by the Rally team;
-3. Customize the [package.json](./package.json) file. At a bare minimum, change the `name`,`description`, `version`, `author`, `repository`, `bugs`, and `homepage` properties;
-4. Provide the encryption data to the `Rally` class constructor in the [src/background.js](./src/background.js) file:
-
-```js
-rally.initialize(
-  // A sample key id used for encrypting data.
-  "sample-invalid-key-id",
-  // A sample *valid* JWK object for the encryption.
-  {
-    "kty":"EC",
-    "crv":"P-256",
-    "x":"f83OJ3D2xF1Bg8vub9tLe1gHMzV76e8Tus9uPHvRVEU",
-    "y":"x_FEzRu9m36HLN_tue659LNpXW6pCyStikYjKIWI5a0",
-    "kid":"Public key used in JWS spec Appendix A.3 example"
-  }
-);
-```
+# Rally Study 01 (RS01)
 
-4. From the forked repository directory, install all the NPM dependencies:
+_status: in development_
 
-```bash
-fork-dir> npm install .
-```
+An exploratory study on what sites people use and how long they use them.
 
-5. Test the customized study in Firefox and commit your changes: doing this right after the initial fork provides a nice and clean cutting point with the original repository, which will simplify future updates (if needed).
+## Requirements
+* [Node.js](https://nodejs.org/en/)
+* [Mozilla web-ext](https://extensionworkshop.com/documentation/develop/getting-started-with-web-ext/)
 
-## Building upon this template
-This template uses [rollup.js](https://rollupjs.org/) as a module bundler and NPM for dependency management. The [manifest.json](./manifest.json) file already includes a sample background scripts.
+This repository contains the code needed to build the Rally Study 01 Web Extension. 
+The study submits data through [Mozilla Rally's Web Extension](https://github.com/mozilla-rally/rally-core-addon),
+or can be run locally without Mozilla Rally installed on your computer.
 
-Dependencies can be added using the [`npm install`](https://docs.npmjs.com/cli/v6/commands/npm-install) command, using the appropriate `--save-dev` or `--save-prod` switch. This command will take care of automatically updating the [package.json](./package.json) file.
+## Quickstart
 
-New modules can be added in the `src/` directory. Modules need to expose the exported functions using the `module.exports` syntax. For example, a class can be exported as follows in a `MyClass.js` file:
+```bash
+# install all dependencies
+npm install
 
-```js
-module.exports = class MyClass {
-  myFunc() {
-    console.log("Testing!");
-  }
-}
-```
+# run developer mode.
+# this will run web-ext and listen for all input files, 
+# and will rebuild and auto-reload for you.
+npm run watch
 
-And then be imported in another file with `const MyClass = require("./MyClass.js");`.
+# build the addon and output the xpi so that it can be side-loaded in Firefox Nightly.
+# See the "running thiis study to collect your own data" 
```

Comment:
removed this section, since I don't think it's relevant anymore https://github.com/mozilla-rally/rally-study-01/pull/73/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L63